flake.nix

```diff
@@ -0,0 +1,59 @@
+{
+  description = " A fast usermode x86 and x86-64 emulator for Arm64 Linux";
+
+  inputs = {
+    nixpkgs.url = "github:nixos/nixpkgs?ref=nixos-unstable";
+    flake-utils.url = "github:numtide/flake-utils";
+  };
+
+  outputs = { self, nixpkgs, flake-utils, ... }:
+    let
+      supportedSystems = [ "x86_64-linux" "aarch64-linux" ];
+      forAllSystems = nixpkgs.lib.genAttrs supportedSystems;
+      nixpkgsFor = forAllSystems (system: import nixpkgs { inherit system; });
+
+    in {
+      packages = forAllSystems (system:
+        let
+          pkgs = nixpkgsFor.${system};
+          deps = with pkgs; [
+            git
+            cmake
+            ninja
+            clang
+            binutils
+            lld
+            llvm
+            SDL2
+            libepoxy
+            nasm
+            pkg-config
+            python3
+            gcc-unwrapped
+            python311Packages.setuptools
+          ];
+        in {
+          fex = pkgs.stdenv.mkDerivation rec {
+            name = "FEXBash";
```

Comment:
Is this the package name? FEXBash is just one utility application. If anything it should match our PPA package name of `fex-emu` https://github.com/FEX-Emu/FEX-ppa/blob/main/deb_base/control#L57